20180224_kubernetes/index.html

```diff
@@ -0,0 +1,223 @@
+            This brought forth 3 improvements: Cost reduction, faster execution speed, and restricted risk (remove errors and security violations).<br />
+        </div>
+    </div>
+
+    <div id="history_5" class="step slide" data-x="2000" data-y="-2300">
+        <q>Enter the world of containers</q>
+        <img src="static/history_5_docker.jpg" />
+        <div class="notes">
+            Docker is a technology providing containers (or OS-level virtualization).
+        </div>
+    </div>
+
+    <div id="history_6" class="step slide text_align" data-x="1000" data-y="-2300">
+        <p>A strong ecosystem was built around Docker</p>
+        <img src="static/history_6.png" />
+        <div class="notes">
+            Prometheus, Docker compose, CoreOS, Mesosphere, Flocker, Cloud 66
+            Helios, Zookeeper, Drone, Twistlock, Weave cloud, Kubernetes
+        </div>
+    </div>
+
+    <div id="history_7" class="step slide text_align" data-x="0" data-y="-2300">
+        <p>And we'll dive into today's most popular orchestration engine:</p>
+        <img src="static/history_7.png" />
+        <div class="notes">
+            <ul>
+                <li>Created by Google, heavily influenced by Google’s Borg system, announced in mid-2014</li>
+                <li>Kubernetes v1 released on July 21, 2015. Google partnered with the Linux Foundation to form the Cloud Native Computing Foundation (CNCF)</li>
+                <li>Now used by Rancher Labs, Red Hat, CoreOS, IBM and Oracle.</li>
+                <li>Became the de-facto container deployment system</li>
+            </ul>
+        </div>
+    </div>
+
+
+    <div id="title" class="step" data-x="0" data-y="0" data-scale="4">
+        <span class="try">What exactly is</span>
+        <h1>Kubernetes</h1>
+        <img src="static/title_interrogative.png" />
+    </div>
+
+    <div id="pods" class="step" data-x="850" data-y="3500" data-rotate="90" data-scale="5">
+        <p>
+        It has sets of <strong>containers</strong> like Docker called <strong>Pods</strong>
+        </p>
+        <img src="static/pods.svg" />
+        <div class="notes">
+            Group of containers that will be co-located on the same host machine and share the same resources and IP address.<br />
+            A pod can define a volume (local disk directory or network disk)<br />
+        </div>
+    </div>
+
+    <div id="nodes" class="step" data-x="3700" data-y="2800" data-rotate="180" data-scale="5">
+        <p>Pods run on <b>Nodes</b> <span class="thoughts">(worker machine)</span></p>
+        <img src="static/nodes.svg" />
+        <div class="notes">
+            A node is a worker machine in Kubernetes and may be a VM or physical machine, depending on the cluster.<br />
+            You might have heard of master/minions before, and nodes were previously called minions.
+            Multiple Pods can run on one Node.<br />
+            Every Kubernetes Node runs at least:
```

Comment:
Succinct version:

_Node_ Vocabulary:
- _Kubelet_: Handles communication & manages _Pod_'s
- _Container Runtime:_ Docker, rkt to run apps